trunk/SUAVE/Methods/Aerodynamics/Airfoil_Panel_Method/airfoil_analysis.py

```diff
@@ -73,18 +73,14 @@ def airfoil_analysis(airfoil_geometry,alpha,Re_L,initial_momentum_thickness=1E-5
                         
     Properties Used:
     N/A
-    """    
-    
-    nalpha       = len(alpha[0,:])
-    nRe          = len(Re_L[0,:]) 
-    nRe_cpts     = len(Re_L)  
-    ncases       = nalpha 
-    ncpts        = nRe_cpts
+    """     
+    ncases       = len(alpha[0,:]) 
+    ncpts        = len(Re_L)  
     x_coord      = airfoil_geometry.x_coordinates
     y_coord      = airfoil_geometry.y_coordinates
     npanel       = len(x_coord)-1 
          
-    if (nalpha !=  nRe):
+    if (ncases !=  len(Re_L[0,:])  ):
```

Comment:
I didn't mean get rid of nRe, I meant get rid of nalpha and nRE_cpts

Reply:
There was redundancy either way so less lines of code